src/io.d

```diff
@@ -149,6 +176,19 @@ unittest
     assert(position.toString == expected);
 }
 
+@("convert position to offset")
```

Comment:
@("convert offset position to string")

Reply:
Was too late in the evening to get it right. :smiley: